layouts/blog/single-sidebar.html

```diff
@@ -24,5 +24,9 @@ <h1 class="f1 lh-solid measure-narrow mb3 fw4">{{ .Title }}</h1>
   {{ partial "shared/sidebar-scaffold.html" . }}
   {{ .Scratch.Set "details" "open" }}
   {{ partial "shared/post-details.html" . }}
+  <!--insert ToC-->
```

Comment:
Yes! Mostly a stylistic decision and thinking ahead to the **single** layout. Let's see if I can do a good job explaining
1. I wanted to be able to use the `post-toc.html` partial for both the **single-sidebar** and **single** post layouts _and_ reuse as much CSS as possible, including the [`.page-sidebar`](https://github.com/spcanelon/hugo-apero/blob/post-toc/assets/scaffold.scss#L153-L163) and  [`.sticky`](https://github.com/spcanelon/hugo-apero/blob/post-toc/assets/scaffold.scss#L403-L408) classes
2. `page-sidebar` used in the aside surrounding the ToC in the **single** layout gives it a background like the one used for **single-sidebar**. This aside needs slightly different styling so that there isn't as much padding on the top (see screenshot A). Modifying the padding for the **single** aside means adjusting the padding for the aside surrounding the entire sidebar in **single-sidebar**, so I gave the **single-sidebar** ToC its own aside instead.

The ToC for the single layout could, of course, not be styled using `page-sidebar` in which case the aside inside an aside isn't necessary. One example would be screenshot B.

Screenshot A: No aside inside an aside, using the `page-sidebar` CSS class

<img width="1003" alt="ToC in the single layout styled the same was the sidebar in single-sidebar, with a background container of a different color" src="https://user-images.githubusercontent.com/49913337/119407458-40a9c300-bcb2-11eb-9712-c5eade394970.png">


Screenshot B: No aside inside an aside, without using the `page-sidebar` CSS class

<img width="968" alt="ToC in the single layout styled more simply with the CSS styling links and no background like that used in the single-sidebar layout" src="https://user-images.githubusercontent.com/49913337/119407310-0cce9d80-bcb2-11eb-8fb0-b076ac9e7a88.png">